App/Redux/PhotoViewingSelectors.ts

```diff
@@ -0,0 +1,25 @@
+import { RootState } from './Types'
+import { ThreadData } from './PhotoViewingRedux'
+
+export function defaultThreadData (state: RootState): ThreadData | undefined {
```

Comment:
Kinda nice to move selectors into a module-wide file. They can get complicated. We should move a lot of the complicated code that exists in our `mapDispatchToProps` into selectors.

Reply:
nice, makes sense